client/rpc-api/src/chain/mod.rs

```diff
@@ -54,6 +54,18 @@ pub trait ChainApi<Number, Hash, Header, SignedBlock> {
 	#[rpc(name = "chain_getFinalizedHead", alias("chain_getFinalisedHead"))]
 	fn finalized_head(&self) -> Result<Hash>;
 
+	/// All head subscription
+	#[pubsub(subscription = "chain_allHead", subscribe, name = "chain_subscribeAllHeads")]
```

Comment:
I'm not a big fan of the subscription name (`chain_allHead`) but don't really have any better ideas.

Reply:
I'm not a fan too, but I didn't want to break backward compatibility and wanted to keep it consistent.

Reply:
IMO it would be better if this method was `chain_newHead` and then the one below was named `chain_bestHead`, but since this rename would be a bit troublesome I also don't have a good suggestion.